flow360/version_check.py

```diff
@@ -0,0 +1,95 @@
+"Client version check module"
+import re
+import sys
+from enum import Enum
+
+import requests
+from packaging.version import Version
+from requests import HTTPError
+
+from .cloud import http_util
+from .solver_version import Flow360Version as sv
+from .version import __version__
+
+
+class VersionSupported(Enum):
+    """
+    If version is supported
+    """
+
+    YES = 1
+    NO = 2
+    CAN_UPGRADE = 3
+
+
+def get_supported_server_versions(app_name):
+    """
+    fetch list of supported versions
+    """
+    try:
+        session = requests.Session()
+        http = http_util.Http(session)
+        response = http.get(f"versions?app_name={app_name}")
+    except HTTPError as exc:
+        raise HTTPError("Error in connecting server") from exc
+
+    versions = [re.sub(r".+-", "", item["version"]) for item in response]
+
+    if len(versions) == 0:
+        raise RuntimeError("Error in fetching supported versions")
```

Comment:
the message should be " the supported versions should not be empty for {app_name}"